auth.py

```
@@ -0,0 +1,195 @@
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from datetime import datetime
+from functools import wraps
+
+from flask import session, jsonify, request
+from models import db, User, OTPToken
+
+
+def generate_otp():
+    """Generate a 6-digit OTP code"""
+    return ''.join([str(random.randint(0, 9)) for _ in range(6)])
```

Ruff (0.14.4)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)